code/game/gamemodes/cult/ritual.dm

```diff
@@ -177,6 +177,58 @@ var/global/list/rnwords = list("ire","ego","nahlizet","certum","veri","jatkaa","
 /obj/effect/rune/proc/check_icon()
 	icon = get_uristrune_cult(word1, word2, word3)
 
+/obj/item/weapon/book/tome/Initialize()
+    . = ..()
+    ritualknife = new /obj/item/weapon/melee/cultknife(src)
+
+/obj/item/weapon/book/tome/verb/verb_manifest_knife()
+	set category = "Object"
+	set name = "Manifest"
+
+	manifest_knife(usr)
+
+/obj/item/weapon/book/tome/proc/manifest_knife(mob/living/carbon/human/user)
+	if(!iscultist(user))
+		user << "<span class='notice'>You're not sure if this is even a manifest.</span>"
+		return
+	else
+		if (!ritualknife)
+			user << "<span class='notice'>\The [src] does not have a ritual dagger in it.</span>"
+			return
+
+		switch (use_check(user, USE_DISALLOW_SILICONS, show_messages = FALSE))

```

Comment:
Why are you explicitly handling the messages here if you only care about success?

Reply:
mostly because this is copy pasted then revised PDA code and I thought that was how to do it